.vitepress/theme/components/PageAuthorComponent.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div v-if="authors.length > 0" class="authors-section">
+    <h2>Page Authors</h2>
+    <div class="page-authors">
+      <div v-for="author in authors" :key="author">
+        <a :href="`https://github.com/${author}`" target="_blank" class="author-link">
+          <img class="author-avatar" :src="`https://github.com/${author}.png`" :alt="author" />
+          <small>{{ author }}</small>
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { onContentUpdated, useData } from 'vitepress';
+import { ref } from 'vue';
+
+const authors = ref([]);
+const pageData = useData();
+
+function refreshData() {
+  authors.value = [];
+
+  if (pageData.frontmatter.value["authors"]) {
+    authors.value = pageData.frontmatter.value["authors"];
+  }
+}
+
+onContentUpdated(() => {
```

Comment:
Is this not statically built? If not whats the reasoning why it has to be done on the client side like this?

Reply:
It is static. This is automatically prerendered during the build stage.